### PR TITLE
added a chapter which describes how to validate against xsd

### PR DESCRIPTION
**Pull request by staabm**
The example code is untested.

### CODE REVIEW

File: xml/reading/index.html [hunk 1]
```diff
@@ -587,6 +587,20 @@ <h2 id="using-element-classes">Using element classes</h2>
...
+
+<pre><code>$reader = new Sabre\Xml\Reader();
+$validXml = $reader-&gt;setSchema('myschema.xsd')
+if ($validXml) {
+    $reader-&gt;xml($xml);
+    print_r($reader-&gt;parse());
+}
+</code></pre>
+
+
```
Comment (Hywan) [inline xml/reading/index.html]: Do we need 4 empty lines?

**Comment by Hywan** [comment]
Sounds good to me. Can you squash your commit into a single one please?

**Comment by staabm** [comment]
@Hywan squashed

**Comment by evert** [comment]
I'm actually just gonna disable the issue system in this REPO. Hopefully others won't make the same mistake anymore.

**Pull request closed without merging**